web/src/Components/TeamDashboard/index.js

```diff
@@ -30,9 +30,8 @@ export default class TeamDashboard extends Component {
     const sortedPlayers = _.sortBy(teamPlayers, (p) => p.salary).reverse();
     const salaries = _.map(sortedPlayers, (p) => p.salary);
     const teamSalary = _.sum(salaries);
-    const salaryCap = calcSalaryCap(allPlayers);
-    const salaryFloor = calcSalaryFloor(allPlayers);
-    const overCap = teamSalary > salaryCap;
+    const { salaryCap, salaryFloor } = calcSalaryLimits(allPlayers);
+    const overCap = teamSalary > salaryCap || teamSalary < salaryFloor;
```

Comment:
We could do a different warning color for under cap

Reply:
Yeah, that's a good idea! I'll write up a couple of salary cap UI updates in an issue.